.eslintrc

```diff
@@ -168,6 +174,7 @@
     },
     "env": {
         "browser": true,
-        "es6": true
+        "es6": true,
+        "node": true
```

Comment:
I suspect you use "node" for the build system ? maybe it's better to use a dedicated .eslintrc in the build/ directory and in every directory that needs it ? My fear is that we may use in an app a variable name that's a global in node but not elsewhere, and eslint does not warn us...